### ISSUE TITLE
Fix missing declaration/export caused by wrong previous edits

### ISSUE THREAD

**Issue by lubekgc** [libical/libical#225]
Fix a typo in the header src/libical/icallangbind.h:
original:
`LIBICAL_ICAL_EXPORT char *callangbind_quote_as_ical_r(const char *str);`
should be:
`LIBICAL_ICAL_EXPORT char *icallangbind_quote_as_ical_r(const char *str);`

**Comment by winterz** [comment]
fixed in the 2.0 and master branches
commit 35b46c0f53cae175ff96dbd982bf582e1331468c

**Comment by ilovezfs** [comment]
@winterz it looks like this snuck into the release tarball, which has
```
LIBICAL_ICAL_EXPORT char *callangbind_quote_as_ical_r(const char *str);
```
in icallangbind.h
